### PR TITLE
DOC: Update README.rst

### PR DESCRIPTION
**Pull request by lboucin**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -72,23 +83,22 @@ the latest release with:
...
-For more information on `Ansys Digital Twins`_, see the `Twin Builder`_
-page on the Ansys website.
 
 .. LINKS AND REFERENCES
-.. _Ansys Digital Twins: https://www.ansys.com/products/digital-twin/
-.. _Twin Builder: https://www.ansys.com/products/digital-twin/ansys-twin-builder
+.. _Digital Twins: https://www.ansys.com/products/digital-twin/
+.. _Ansys Twin Builder: https://www.ansys.com/products/digital-twin/ansys-twin-builder
 .. _Documentation: https://twin.docs.pyansys.com/
 .. _Getting started: https://twin.docs.pyansys.com/release/0.1/getting_started/index.html
```
Comment (Revathyvenugopal162) [inline README.rst:103]:
```suggestion
.. _Getting started: https://twin.docs.pyansys.com/release/0.2/getting_started/index.html
```
Change the version pointing to latest. This is a bug and we will try to keep the latest stable in the next release of pyansys/actions. Till then, you might have to change it manually.

Comment (lboucin) [reply README.rst]: Thanks @Revathyvenugopal162!

File: README.rst [hunk 2]
```diff
@@ -36,8 +36,19 @@ PyTwin
 
 Overview
 --------
-PyTwin is a Python package that eases `Ansys Digital Twins`_ consumption workflows.
+PyTwin is a Python package for consuming a digital twin model that has been exported from
+Ansys Twin Builder as a TWIN file. Each TWIN file is a self-contained app (Twin Runtime)
```
Comment (PipKat) [inline README.rst:40]:
```suggestion
Ansys Twin Builder as a TWIN file. Each TWIN file is a self-contained app (twin runtime)
```

File: README.rst [hunk 3]
```diff
@@ -36,8 +36,19 @@ PyTwin
...
+that consists of two parts:
+
+- The twin model, which is the simulation model created and compiled in Twin Builder
+- The runtime SDK, which is the shared library for consuming the twin model within
+  the deployment workflow.
+
+Because PyTwin wraps a Twin Runtime in a Python interface, you do not need a Twin Builder
```
Comment (PipKat) [inline README.rst:47]:
```suggestion
Because PyTwin wraps a twin runtime in a Python interface, you do not need a Twin Builder
```

**Review by PipKat — APPROVED** [review]

**Review by chrpetre — APPROVED** [review]

**Review by lboucin — COMMENTED** [review]
LGTM. Thanks!

**Pull request merged**